Speech/src/Connection/Rest.php

```diff
@@ -27,8 +27,7 @@
  * Implementation of the
  * [Google Cloud Speech JSON API](https://cloud.google.com/speech/reference/rest/).
  *
- * @deprecated This class is no longer supported and will be removed in a future
- * release.
+ * @internal
```

Comment:
Same here!

Reply:
Missed it in the last commit, thanks for the callout.